### DISCUSSION TITLE
Parameters that are nearly always the same values should not be required to be part of a moveit config

### DISCUSSION THREAD

**Discussion by tylerjw** [moveit/moveit2#2713] [Ideas]
MoveIt has way too many parameters and in most cases, many of the parameters are always the same between MoveIt configs. These parameters should be identified, have defaults set in code, and be removed from the configs generated from MSA.

**Comment by mikeferguson** [comment]
Counterpoint: If you want to change the parameter, the MSA config is often the easiest way to figure out where the parameter should be placed to get loaded properly.

**Comment by gavanderhoorn** [comment]
another counterpoint: defaults are the devil when debugging / diagnosing something.

Especially silent defaults which have no visible presence in configuration files.

Please do not go that way.
